src/app.types.tsx

```diff
@@ -32,36 +32,37 @@ export interface CatalogueCategory {
   is_leaf: boolean;
   catalogue_item_properties?: CatalogueCategoryFormData[];
 }
+
 export interface AddManufacturer {
   name: string;
   url?: string;
-  address: Address | undefined;
+  address?: Address;
   telephone?: string;
 }
 
 export interface AddManufacturerResponse {
```

Comment:
the Manufacturer type has an ID though?

Reply:
Its the same as https://github.com/ral-facilities/inventory-management-system/pull/112#discussion_r1387925427, it is already returning an id but it wont error if its attempting to cast to this type, it will just ignore it so changing to Manufacturer will just pick it up again.